src/Rings/AbelianClosure.jl

```diff
@@ -1084,6 +1119,23 @@ Base.conj(elm::QQAbFieldElem) = elm^QQAbAutomorphism(-1)
 
 Base.isreal(elm::QQAbFieldElem) = conj(elm) == elm
 
+# compare real `QQAbFieldElem`s
+function _isless_via_qqbar(a, b)
+  F = QQBarField()
+  return Base.isless(F(a), F(b))
+end
+
+Base.isless(a::QQAbFieldElem, b::QQAbFieldElem) = _isless_via_qqbar(a, b)
+Base.isless(a::QQAbFieldElem, b::ZZRingElem) = _isless_via_qqbar(a, b)
+Base.isless(a::QQAbFieldElem, b::QQFieldElem) = _isless_via_qqbar(a, b)
+Base.isless(a::QQAbFieldElem, b::Int) = _isless_via_qqbar(a, b)
+Base.isless(a::QQFieldElem, b::QQAbFieldElem) = _isless_via_qqbar(a, b)
+Base.isless(a::ZZRingElem, b::QQAbFieldElem) = _isless_via_qqbar(a, b)
+Base.isless(a::Int, b::QQAbFieldElem) = _isless_via_qqbar(a, b)
```

Comment:
Comparison with `Rational` and `Integer` would also be nice. At this point might use `@eval` to keep it readable (unfortunately we can't use install for `RationalUnion` to avoid ambiguities)
```suggestion
for T in (QQFieldElem, ZZRingElem, Int, Integer, Rational)
  @eval begin
    Base.isless(a::QQAbFieldElem, b::$T) = _isless_via_qqbar(a, b)
    Base.isless(a::$T, b::QQAbFieldElem) = _isless_via_qqbar(a, b)
  end
end
```